File: 프로그래머스/1/87389. 나머지가 1이 되는 수 찾기/나머지가 1이 되는 수 찾기.cpp
#include <string>
#include <vector>

using namespace std;

int solution(int n) {
    int answer = 0;
    bool check = true;
    for(int i=1; i<=n; i++) {
        if(n % i == 1 && check) answer = i, check = false;
    }
    return answer;
}

File: 프로그래머스/1/87389. 나머지가 1이 되는 수 찾기/나머지가 1이 되는 수 찾기.swift
import Foundation

func solution(_ n:Int) -> Int {
    var check: Bool = false
    var answer: Int = 1;
    
    while !check {
        if n % answer == 1 {
            check = true
        } else {
            answer += 1
        }
    }
    
    return answer
}